src/services/postService.js

```diff
@@ -2,8 +2,8 @@ const uuid = require("uuid");
 const { throwIfError } = require('../utilities/dynamoUtilities');
 const postDAO = require("../repository/postDAO");
 
-async function createPost(userID, description, score, title){
-    const post = {class: "post", itemID: uuid.v4(), postedBy: userID, description, score, title, replies: [], likedBy: [], isFlagged: 0};
+async function createPost(userID, description, score, title, tags){
+    const post = {class: "post", itemID: uuid.v4(), postedBy: userID, description, score, title, replies: [], likedBy: [], tags: tags ? tags.split(',') : [], isFlagged: 0};
```

Comment:
I'm not sure if you have the tag as a string in the body or not, but I believe .split is a string thing only. Does JSON allow for arrays to be sent through? It should be easier that way if it does. Also you will have to input validate this so you can assure you get a string/array because if they send up 4 for tags then tags will just be 4

Reply:
Good point, I will turn tags into a list input